desktop/main.cpp

```diff
@@ -238,7 +237,12 @@ int main(int argc, char **argv) {
   QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
   QApplication app(argc, argv);
 
-  QCoreApplication::setApplicationName("Status");
+  QCoreApplication::setOrganizationName("Status");
+  QCoreApplication::setOrganizationDomain("status.im");
+  QCoreApplication::setApplicationName("StatusDesktop");
+
+  // Init AppConfig
+  AppConfig::inst().getValue("logging_enabled");
```

Comment:
Can't we use the `LOGGING_ENABLED` constant here?

Reply:
Yes we can, fixed.